pages/models/generations/SM1/index.md

```diff
@@ -0,0 +1,170 @@
+It can be round rotor, commonly used in high rotation speed application, or salient pole, used in lower speed applications (i.e. hydro generation) where multiple pair of poles are needed to work at the grid frequency.[[2]](#2) The windings axis will be pointing in a perpendicular direction to the turbine axis, and will rotate with the rotor. This winding will have a current generated by the exciter that will produce a rotating magnetic field ($$\phi_f$$). 
+
+#### Exciter
+
+The exciter is the source of power that generates the current flowing through the rotor winding. It applies a DC voltage $$E_f$$ that generates a current in the rotor winding, creating an electromagnet. 
+
+#### Stator
+
+It is the static part of the machine, as its name indicates. This structure gives physical support to the whole machine, while it tipically contains the armature winding. It is a three-phase winding distributed in a circular cavity 120º apart one from each other. The uniform rotation of the magnetic field created by the rotor winding will generate currents at each winding 120º apart in the phasorial space. The instantaneous value for each phase is noted as $$i_a$$, $$i_b$$, and $$i_c$$[[2]](#2).
+
+<p align="center">
+<img src="{{'/pages/models/generations/SM1/SM_rot_stat.svg' | relative_url }}"
+     alt="Operation of a Synchronous Machine schematic"
+     style="float: center; width: 400px;" />
+
+</p>
+
+#### Governor
+
+The governor is the controller of the speed of the motor. It will ensure that the frequency of the rotation is maintained at the setpoint, which is the grid frequency.
+The control will be done by changing the amount of steam entering in the turbine, increasing or decreasing the rate of steam depending on the error signal. 
+
+### Operational principles
+
+The dynamic equation that governs the system relates the mechanical torque applied by the turbine and the electrical torque applied by the grid[[3]](#3):
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$J\frac{2}{p_f}  \frac{d\omega}{dt} = T_m - T_e - T_{fw}$$
+</div>
+
+where $$J$$ is the inertia of the rotating body, $$T_m$$ is the mechanical torque, $$T_e$$ is the electrical torque and $$T_{fw}$$ is the friction and windage torque.
+
+Regarding the electrical part of the model, the rotor winding is magnetized by the exciter current, creating a magnetic field that will rotate with the rotor's angular velocity. As it spins, the stator windings will be subjected to a variable magnetic flux caused by the rotation of the produced magnetic field which, according to Faraday's Law, will induce a current in the windings.
+
+The rotational speed of the rotor of a syncrhonous machine is proportional to the grid frequency times the number of poles (i.e. number of different three-phase windings). The induced field $$e_f$$ will lead the terminal voltage $$v_t$$ by a load angle denoted as $$\delta$$.
```

Comment:
can you explain the convention you use here for dq axis and delta definition?
see slide 5 here: https://www.powerworld.com/files/Synchronous-Machines.pdf

Reply:
Convention used added in Figure 4, line 235